easybuild/tools/build_log.py

```diff
@@ -63,7 +64,19 @@ class EasyBuildError(LoggedException):
 
     def __init__(self, msg, *args):
         """Constructor: initialise EasyBuildError instance."""
-        msg = msg % args
+        # figure out where error was raised from
+        # current frame: this constructor, one frame above: location where this EasyBuildError was created/raised
+        frameinfo = inspect.getouterframes(inspect.currentframe())[1]
+
+        # determine short location of Python module where error was raised from (starting with 'easybuild/')
+        path_parts = frameinfo[1].split(os.path.sep)
+        relpath = path_parts.pop()
+        while not (relpath.startswith('easybuild/') or relpath.startswith('vsc/')) and path_parts:
+            relpath = os.path.join(path_parts.pop() or os.path.sep, relpath)
```

Comment:
I'm not 100% happy with this, but couldn't find a better way after trying for a while.

Code golf: only retain the interesting part from something like `/Users/kehoste/work/easybuild-framework/easybuild/tools/filetools.py`, i.e. the `easybuild/tools/filetools.py` part.